joint_trajectory_controller/src/trajectory.cpp

```diff
@@ -138,11 +167,15 @@ bool Trajectory::sample(
       // Do interpolation
       else
       {
-        // it changes points only if position and velocity do not exist, but their derivatives
+        // it changes points only if position and velocity are not exist, but their derivatives
```

Comment:
Seems like the grammar was correct previously?

```suggestion
        // it changes points only if position and velocity do not exist, but their derivatives
```